appveyor.yml

```diff
@@ -12,7 +12,7 @@ environment:
 install:
   - ps: Install-Product node $env:nodejs_version
   - set CI=true
-  - npm install --global npm@latest
+  - npm install --global npm@4
```

Comment:
If you think it's important, I'm suggesting to just remove `npm install --global npm`.
So appveyor will use the default version

Reply:
Or if you prefer using yarn here, just go with it.. But i think we're gonna lose our chance to catch *npm bugs* on Windows that may affect even more users than yarn